web/Dockerfile

```diff
@@ -32,6 +32,10 @@ RUN pnpm install --frozen-lockfile
 ENV DOCKER_BUILD 1
 ENV NEXT_MANUAL_SIG_HANDLE true
 
+# set environment variables 
+ARG PORTER_NEXT_PUBLIC_BUILD_ID
+ENV NEXT_PUBLIC_BUILD_ID=$PORTER_NEXT_PUBLIC_BUILD_ID
```

Comment:
this will not work, We should expose an ARG `NEXT_PUBLIC_BUILD_ID`. Based on the porter docs I'd assume that they will pass the env without the prefix